assets/Mods/shared/UEHelpers/UEHelpers.lua

```diff
@@ -28,68 +48,210 @@ function UEHelpers.GetUEHelpersVersion()
----@return AActor
+--- Any UObject that has a GetWorld() function can be used as WorldContext.
+---@return UObject
 function UEHelpers.GetWorldContextObject()
     return UEHelpers.GetPlayerController()
 end
 
+---Returns hit actor from FHitResult.<br>
+---The function handles the struct differance between UE4 and UE5
+---@param HitResult FHitResult
+---@return AActor
+function UEHelpers.GetActorFromHitResult(HitResult)
+    if not HitResult or not HitResult:IsValid() then
+        return RemoteObject:new() ---@type AActor
+    end
+    
+    if UnrealVersion:IsBelow(5, 0) then
+        return HitResult.Actor:Get()
+    end
+    return HitResult.HitObjectHandle.Actor:Get()
+end
+
+---@param ForceInvalidateCache boolean? # Force update the cache
+---@return UGameplayStatics
 function UEHelpers.GetGameplayStatics(ForceInvalidateCache)
+    ---@type UGameplayStatics
     return CacheDefaultObject("/Script/Engine.Default__GameplayStatics", "UEHelpers_GameplayStatics", ForceInvalidateCache)
 end
 
+---@param ForceInvalidateCache boolean? # Force update the cache
+---@return UKismetSystemLibrary
 function UEHelpers.GetKismetSystemLibrary(ForceInvalidateCache)
+    ---@type UKismetSystemLibrary
     return CacheDefaultObject("/Script/Engine.Default__KismetSystemLibrary", "UEHelpers_KismetSystemLibrary", ForceInvalidateCache)
 end
 
+---@param ForceInvalidateCache boolean? # Force update the cache
+---@return UKismetMathLibrary
 function UEHelpers.GetKismetMathLibrary(ForceInvalidateCache)
+    ---@type UKismetMathLibrary
     return CacheDefaultObject("/Script/Engine.Default__KismetMathLibrary", "UEHelpers_KismetMathLibrary", ForceInvalidateCache)
 end
 
-function UEHelpers.GetKismetMathLibrary(ForceInvalidateCache)
-    return CacheDefaultObject("/Script/Engine.Default__KismetMathLibrary", "UEHelpers_KismetMathLibrary", ForceInvalidateCache)
+---@param ForceInvalidateCache boolean? # Force update the cache
+---@return UKismetStringLibrary
+function UEHelpers.GetKismetStringLibrary(ForceInvalidateCache)
+    ---@type UKismetStringLibrary
+    return CacheDefaultObject("/Script/Engine.Default__KismetStringLibrary", "UEHelpers_KismetStringLibrary", ForceInvalidateCache)
+end
+
+---@param ForceInvalidateCache boolean? # Force update the cache
+---@return UKismetTextLibrary
+function UEHelpers.GetKismetTextLibrary(ForceInvalidateCache)
+    ---@type UKismetTextLibrary
+    return CacheDefaultObject("/Script/Engine.Default__KismetTextLibrary", "UEHelpers_KismetTextLibrary", ForceInvalidateCache)
 end
 
+---@param ForceInvalidateCache boolean? # Force update the cache
+---@return UGameMapsSettings
 function UEHelpers.GetGameMapsSettings(ForceInvalidateCache)
+    ---@type UGameMapsSettings
     return CacheDefaultObject("/Script/EngineSettings.Default__GameMapsSettings", "UEHelpers_GameMapsSettings", ForceInvalidateCache)
 end
 
+---Returns Empty FName aka. "None"
+---@return FName
+function UEHelpers.FName_None()
+    return NAME_None
+end
```

Comment:
What's the point in having this ? They'll just have to remember this function instead of `NAME_None`.
Shouldn't we tell people to simply use `NAME_None` instead ?
Perhaps we can add `NAME_None` to a few FName related places in the docs.
I like the FindFName/AddFName additions though, at least then people don't have to know about the reversed default of our FName constructor.

Reply:
I just found it easier to remember, to build a memory chain for it in the brain and if you use "[Custom Lua Bindings](https://docs.ue4ss.com/guides/using-custom-lua-bindings.html#using-custom-lua-bindings)" you can get autocomplete for it.  
However, now that think about it again, it's a better idea to add NAME_None to Types.lua